src/opm/input/eclipse/EclipseState/InitConfig/Equil.cpp

```diff
@@ -110,18 +119,21 @@ namespace Opm {
             const auto wet_gas_init = record.getItem<EQUIL::BLACK_OIL_INIT_WG>().get<int>(0) <= 0;
             const auto target_accuracy = record.getItem<EQUIL::OIP_INIT>().get<int>(0);
             const auto humid_gas_init = record.getItem<EQUIL::BLACK_OIL_INIT_HG>().get<int>(0) <= 0;
+            // Currently set the same as live_oil_init?
+            const auto live_water_init = record.getItem<EQUIL::BLACK_OIL_INIT>().get<int>(0) <= 0;
 
```

Comment:
I start adding the infrastructure to store a separate live_water_init parameter from the EQUIL keyword similar to what is done for wet_gas_init, but I am not sure if that is the best solution. Would it be better to just use the BLACK_OIL_INIT item? At the current stage we can not combine liveoil and livewater (i.e. gas in oil and water phase) anyway, and if we could, it may still not make sense to set up a case where RSVD is set not RSWVD or the opposite. Adding yet another item to the EQUIL keyword may not be what we want. Any comments @bska, @plgbrts